content/en/docs/setup/install/helm/index.md

```diff
@@ -166,6 +166,14 @@ installed above.
     $ kubectl delete namespace istio-system
     {{< /text >}}
 
+## Uninstall revision tags
+
+If you decide to rollback to the old control plane, instead of completing the canary upgrade,
+you can uninstall the canary revision using `istioctl x uninstall --revision=canary`.
```

Comment:
Is there a requirement to use `istioctl` here?

Reply:
No. Added `helm template | kubectl delete` instructions